src/frontends/common/src/manager.cpp

```diff
@@ -106,6 +106,12 @@ class FrontEndManager::Impl {
     }
 
     void register_front_end(const std::string& name, const std::string& library_path) {
+#if !defined(__EMSCRIPTEN__) && !defined(__ANDROID__)
+        if (fs::is_symlink(library_path))
```

Comment:
can we define function is_symlink which will return false for cases, when implementation is not available (e.g. Android)

In this case we don't have to use conditional compilation in all places, all conditional code will be in file_utils.cpp